cg/server/dto/delivery_message/delivery_message_response.py

```diff
@@ -3,3 +3,7 @@
 
 class DeliveryMessageResponse(BaseModel):
     message: str
+
+
+class DeliveryMessageOrderResponse(DeliveryMessageResponse):
```

Comment:
I've started to question our naming pattern, what do you think of skipping the `Response` part? If necessary to denote the type of model, maybe we should just add `DTO`.

The data model does not really have anything to do with it ultimately being sent as a payload of a HTTP request. We could reuse the service producing these models in other contexts. I think the `Response` part introduces an implicit coupling to a HTTP response that is potentially misleading and unnecessary

Reply:
Hmm, I do like the "Response" and the implicit coupling 🤔 Might be enough to look at the typed response type though. How would we discern what the request and response is?

I mean these are designed specifically for the endpoints and I do not like them being used for other things. I was even debating extending the `DeliveryMessageResponse` class for this reason.

Reply:
I will leave as is for now, and we could consider changing it in the future.

Reply:
It is unlikely that we would use the delivery message service in a different context, but not the `OrderService` for instance. The data is just data, it is not necessarily supposed to be sent in a http response.

For instance, consider that the service is called in a CLI command. Or inside another service. I think we should drop the `Response` suffix from models in the service layer (we probably can keep it for HTTP API clients).

Feel free to leave it as is, just want to point out this coupling that is misleading.

Reply:
I mean, I would always want a DTO specifically for the requests and responses for endpoints. I guess we could discuss where they should be placed though?

I think it adds value that we do not see them as only data. This is data specifically packaged for this purpose in my mind. We could of course discuss it in a meeting or in slack or in person etc.